rules/auth0_rules/auth0_user_joined_tenant.py

```diff
@@ -0,0 +1,43 @@
+from global_filter_auth0 import filter_include_event
+from panther_auth0_helpers import auth0_alert_context, is_auth0_config_event
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    if not filter_include_event(event):
+        return False
+    data_description = deep_get(event, "data", "description", default="<NO_DATA_DESCRIPTION_FOUND>")
+    scopes = deep_get(
+        event,
+        "data",
+        "details",
+        "request",
+        "auth",
+        "credentials",
+        "scopes",
+        default=["<NO_CREDENTIAL_SCOPE>"],
+    )
+    state = deep_get(event, "data", "details", "request", "body", "state", default="<NO_STATE>")
+    return all(
+        [
+            data_description == "Update an invitation",
+            "update:tenant_invitations" in scopes,
```

Comment:
in this event there should only be one value in scopes, we should check for a direct match of scopes [0] == "update:tenant_invitations" as direct matches are favored over 'in' or 'contains'